piet-gpu/shader/coarse.comp

```diff
@@ -451,5 +459,8 @@ void main() {
     }
     if (bin_tile_x + tile_x < conf.width_in_tiles && bin_tile_y + tile_y < conf.height_in_tiles) {
         Cmd_End_write(cmd_alloc, cmd_ref);
+        if (max_blend_depth > BLEND_STACK_SPLIT) {
```

Comment:
Oops. I am surprised my tests worked with this still in place. I guess I was lucky that the memory happened to be accessible.